.changeset/eleven-bats-tease.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes-backend': minor
+---
+
+PermissionApi of type PermissionEvaluator must now be passed as a parameter when calling proxy.createRequestHandler() for the KubernetesProxyEndpoint. Kubernetes `/proxy` endpoint now requires two tokens in its header per request.The field `X-Kubernetes-Authentication` needs the value of a authentication authorities' bearer token.The field `Authorization` should contain a backstage identity token.
```

Comment:
Two things here:
1. I don't think anybody actually calls `proxy.createRequestHandler` in the wild; the most important thing to mention is that `KubernetesBuilder.createBuilder` now requires the field `permissions` of type `PermissionEvaluator`. This sis where integrators will feel a difference. In fact, the installation docs need to be updated:
    https://github.com/backstage/backstage/blob/5673da2bcb43ef3330c67c418e37074af926ba06/docs/features/kubernetes/installation.md?plain=1#L71-L75

1. For whatever reason, in my experience folks tend to talk about a request having plural _headers_, rather than a request's _header_ having plural fields.
    ```suggestion
    `KubernetesBuilder.create` now requires the a `PermissionEvaluator`. The `/proxy` endpoint now requires two tokens: the `X-Kubernetes-Authorization` header should contain a bearer token for the target cluster, and the `Authorization` header should contain a backstage identity token.
    ```